docs/source/auth/index.rst

```diff
@@ -612,6 +612,23 @@ Authentication configuration is located at ``mlflow/server/auth/basic_auth.ini``
      - Default admin username if the admin is not already created
    * - ``admin_password``
      - Default admin password if the admin is not already created
+   * - ``authorization_function``
+     - Function to authenticate requests
+
+The ``authorization_function`` setting supports pluggable authentication methods
+if you want to use another authentication method than HTTP basic auth. The value
+specifies ``module_name:function_name``. The function has the following signature:
+
+    .. code-block:: python
+
+        def authenticate_request() -> Union[Authorization, Response]:
+            ...
+
+The function should return an ``Authorization`` object if the request is authenticated,
```

Comment:
```suggestion
The function should return an ``werkzeug.datastructures.Authorization`` object if the request is authenticated,
```

to be more precise.

Reply:
I'll make this change manually. By adding `werkzeug`, we'll need to switch from "an" to "a" as well.